NativeCode/AudioPluginUtil.cpp

```diff
@@ -257,6 +261,7 @@ void HistoryBuffer::ReadBuffer(float* buffer, int numsamplesTarget, int numsampl
     buffer[numsamplesTarget] = (float)n; // how many samples were written
 }
 
+#ifndef PLATFORM_MUTEX
```

Comment:
Maybe we should extract `Mutex` to its own source files so we don't have so many ifdef blocks

Reply:
Okay, I'll split the mutex class. (It's better to customize it, especially platform dependent.)

Reply:
Okay, I've revertes changes for PLATFORM_MUTEX.